cmd/main.go

```diff
@@ -16,13 +17,20 @@ func main() {
 		log.Fatal(err)
 	}
 
-	dbUser := os.Getenv("DB_USER")
-	dbPassword := os.Getenv("DB_PASSWORD")
-	dbName := os.Getenv("DB_NAME")
-	db := conn.Connect(dbUser, dbPassword, dbName)
+	port, err := strconv.Atoi(os.Getenv("DB_PORT"))
+	if err != nil {
+		log.Fatal("port should be an integer")
+	}
 
+	cfg := conn.DefaultConfig(os.Getenv("DB_NAME")).
```

Comment:
Why don't we move this over to [db.go](https://github.com/somos831/somos-backend/blob/main/db/db.go), that way all of the database connection logic can be in one place and that would also eliminate the need for the setters. From main we would just call `Connect()`.